app.py

```
@@ -960,6 +995,109 @@ async def update_message_rating():
+                    headers = {
+                        "Content-Type": "application/json",
+                        "Accept": "application/json",
+                        "Authorization": f"Bearer {app_settings.n8n.bearer_token}",
+                    }
+                    
+                    timeout_ms = app_settings.n8n.timeout_ms if app_settings.n8n else 15000
+                    async with httpx.AsyncClient(timeout=float(timeout_ms) / 1000) as http_client:
+                        webhook_response = await http_client.post(app_settings.n8n.webhook_url, json=n8n_payload, headers=headers)
+                        if webhook_response.status_code >= 400:
+                            logging.warning(f"n8n webhook for rating update returned status {webhook_response.status_code}")
+            except Exception as webhook_error:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)